x-pack/plugins/osquery/server/handlers/action/create_action_handler.ts

```diff
@@ -124,19 +124,14 @@ export const createActionHandler = async (
           type: 'INPUT_ACTION',
           input_type: 'osquery',
           agents: query.agents,
-          user_id: metadata?.currentUser,
+          user_id: metadata?.currentUser || 'osquery',
```

Comment:
Not sure what should be the default user here when the current user is not available? I assume this is the case for automated actions? @tomsonpl

Reply:
If I remember correctly, we do depend on 'undefined' value so we do not fetch automated actions on our osquery list. I don't think changing this to 'osquery' make much sense, since all the actions are osquery actions ;p Does leaving this at it is bring any TS problems?

Reply:
Yeah, the `user_id` should be a `string`. I think we can use an empty string here instead.